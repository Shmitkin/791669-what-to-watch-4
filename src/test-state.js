export const movie = {
  title: `Pulp Fiction`,
  videoPrev: `img/seven-years-in-tibet.mp4`,
  preview: `img/pulp-fiction.jpg`,
  poster: `img/pulp-fiction.jpg`,
  background: `img/pulp-fiction.jpg`,
  genre: `Drama`,
  description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent varius quis lorem id iaculis. Sed eget luctus orci. Donec cursus lectus auctor quam tempus ultricies. Nullam at aliquam massa. Donec mollis quis urna eget condimentum. Mauris at dictum ante. Phasellus maximus massa augue, ut luctus dolor rutrum et. Aliquam faucibus turpis lectus, sed hendrerit dolor malesuada at.`,
  rating: 2.3,
  votes: 293,
  director: `Best Director`,
  starring: `Nicola Cage, Cara Delvin, Joseph Moser`,
  duration: 87,
  release: 2020,
};

export const movies = [
  {
    title: `Seven Years IN Tibet`,
    videoPrev: `img/seven-years-in-tibet.mp4`,
    preview: `img/seven-years-in-tibet.jpg`,
    poster: `img/seven-years-in-tibet.jpg`,
    background: `img/seven-years-in-tibet.jpg`,
    genre: `Family`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent varius quis lorem id iaculis. Sed eget luctus orci. Donec cursus lectus auctor quam tempus ultricies. Nullam at aliquam massa. Donec mollis quis urna eget condimentum. Mauris at dictum ante. Phasellus maximus massa augue, ut luctus dolor rutrum et. Aliquam faucibus turpis lectus, sed hendrerit dolor malesuada at.`,
    rating: 5.0,
    votes: 1255,
    director: `Best Director`,
    starring: `Nicola Cage, Cara Delvin, Joseph Moser`,
    duration: 24,
    release: 2005,
  },
  {
    title: `Moonrise Kingdom`,
    videoPrev: `img/seven-years-in-tibet.mp4`,
    preview: `img/moonrise-kingdom.jpg`,
    poster: `img/moonrise-kingdom.jpg`,
    background: `img/moonrise-kingdom.jpg`,
    genre: `Horror`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent varius quis lorem id iaculis. Sed eget luctus orci. Donec cursus lectus auctor quam tempus ultricies. Nullam at aliquam massa. Donec mollis quis urna eget condimentum. Mauris at dictum ante. Phasellus maximus massa augue, ut luctus dolor rutrum et. Aliquam faucibus turpis lectus, sed hendrerit dolor malesuada at.`,
    rating: 4.8,
    votes: 213,
    director: `Best Director`,
    starring: `Nicola Cage, Cara Delvin, Joseph Moser`,
    duration: 79,
    release: 2012,
  },
  {
    title: `Snatch`,
    videoPrev: `img/seven-years-in-tibet.mp4`,
    preview: `img/snatch.jpg`,
    poster: `img/snatch.jpg`,
    background: `img/snatch.jpg`,
    genre: `Comedy`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent varius quis lorem id iaculis. Sed eget luctus orci. Donec cursus lectus auctor quam tempus ultricies. Nullam at aliquam massa. Donec mollis quis urna eget condimentum. Mauris at dictum ante. Phasellus maximus massa augue, ut luctus dolor rutrum et. Aliquam faucibus turpis lectus, sed hendrerit dolor malesuada at.`,
    rating: 3.0,
    votes: 22,
    director: `Best Director`,
    starring: `Nicola Cage, Cara Delvin, Joseph Moser`,
    duration: 210,
    release: 1998,
  },
];

export const genres = [
  `First`,
  `Second`,
  `Third`,
  `Fourth`,
  `Fifth`
];
