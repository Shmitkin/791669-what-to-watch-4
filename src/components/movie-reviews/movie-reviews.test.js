import React from "react";
import renderer from "react-test-renderer";

import MovieReviews from "./movie-reviews.jsx";
import {comments} from "../../test-state.js";


it(`Should MovieReview render correctly`, () => {

  const tree = renderer
    .create(
        <MovieReviews
          reviews = {comments}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
