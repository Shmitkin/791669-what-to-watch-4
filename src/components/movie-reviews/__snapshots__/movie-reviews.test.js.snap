// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MovieReview render correctly 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          comment text
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            author1
          </cite>
          <time
            className="review__date"
            dateTime="2020-06-16T10:54:44.818Z"
          >
            June 16, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        7
      </div>
    </div>
  </div>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          comment text2
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            author2
          </cite>
          <time
            className="review__date"
            dateTime="2020-06-16T11:54:44.818Z"
          >
            June 16, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        9
      </div>
    </div>
  </div>
</div>
`;
